webrtc.html

```diff
@@ -1272,6 +1272,13 @@ <h4>Set the RTCSessionDescription</h4>
                     <p>If <var>connection</var>.<a>[[\IsClosed]]</a> is
                     <code>true</code>, then abort these steps.</p>
                   </li>
+                  <li>
+                    <p>If any {{RTCPeerConnection/addTrack()}} methods succeeded
+                    during the process to apply <var>description</var>, abort
+                    these steps and start the process over as if they had
+                    succeeded prior, to include the extra transceiver(s) in the
+                    process.</p>
```

Comment:
OK so we don't roll anything back, we just apply the process again? The end result would be the same as if what we did from the start was "await SRD; addTrack; await SRD;"?

Reply:
Makes sense to me

Reply:
> OK so we don't roll anything back, we just apply the process again?

Right, where _"process"_ = [section 5.6.](https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-25#section-5.6) in JSEP.

> end result would be the same as if what we did from the start was "SRD; addTrack; await SRD;"?

Better. We're talking a subset of that, since that would run both JSEP section 5.6. **_and_** the SRD queued success steps in webrtc-pc twice, with side-effects like events firing twice. We wouldn't be firing events twice. But broadly speaking yes.